src/test/java/net/snowflake/ingest/streaming/internal/StreamingIngestBigFilesIT.java

```diff
@@ -0,0 +1,199 @@
+package net.snowflake.ingest.streaming.internal;
+
+import static net.snowflake.ingest.TestUtils.verifyTableRowCount;
+import static net.snowflake.ingest.utils.Constants.ROLE;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.*;
+import java.util.concurrent.*;
+import net.snowflake.ingest.TestUtils;
+import net.snowflake.ingest.streaming.OpenChannelRequest;
+import net.snowflake.ingest.streaming.SnowflakeStreamingIngestChannel;
+import net.snowflake.ingest.streaming.SnowflakeStreamingIngestClientFactory;
+import net.snowflake.ingest.utils.Constants;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+/** Ingest large amount of rows. */
+@RunWith(Parameterized.class)
+public class StreamingIngestBigFilesIT {
+  @Parameterized.Parameters(name = "{0}")
+  public static Collection<Object[]> bdecVersion() {
+    return TestUtils.getBdecVersionItCases();
+  }
+
+  private static final String TEST_DB_PREFIX = "STREAMING_INGEST_TEST_DB";
+  private static final String TEST_SCHEMA = "STREAMING_INGEST_TEST_SCHEMA";
+
+  private Properties prop;
+
+  private SnowflakeStreamingIngestClientInternal<?> client;
+  private Connection jdbcConnection;
+  private String testDb;
+
+  private final Constants.BdecVersion bdecVersion;
+
+  public StreamingIngestBigFilesIT(
+      @SuppressWarnings("unused") String name, Constants.BdecVersion bdecVersion) {
+    this.bdecVersion = bdecVersion;
+  }
+
+  @Before
+  public void beforeAll() throws Exception {
+    testDb = TEST_DB_PREFIX + "_" + UUID.randomUUID().toString().substring(0, 4);
+    // Create a streaming ingest client
+    jdbcConnection = TestUtils.getConnection(true);
+
+    jdbcConnection
+        .createStatement()
+        .execute(String.format("create or replace database %s;", testDb));
+    jdbcConnection
+        .createStatement()
+        .execute(String.format("create or replace schema %s.%s;", testDb, TEST_SCHEMA));
+    // Set timezone to UTC
+    jdbcConnection.createStatement().execute("alter session set timezone = 'UTC';");
+    jdbcConnection
+        .createStatement()
+        .execute(String.format("use warehouse %s", TestUtils.getWarehouse()));
+
+    prop = TestUtils.getProperties(bdecVersion);
+    if (prop.getProperty(ROLE).equals("DEFAULT_ROLE")) {
+      prop.setProperty(ROLE, "ACCOUNTADMIN");
+    }
+    client =
+        (SnowflakeStreamingIngestClientInternal<?>)
+            SnowflakeStreamingIngestClientFactory.builder("client1").setProperties(prop).build();
```

Comment:
can this code be de-duped with `StreamingIngestIT`?

Reply:
sure, done.

Reply:
`StreamingIngestIT` has another create database that this one doesn't have. Also not everything in the `beforeAll` can be extracted in one method, just four statements (create database, schema, set session tz and warehouse) could be de-duped, I don't think that will help much.